lms/djangoapps/api_manager/users/serializers.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, *args, **kwargs):
                 self.fields.pop(field_name)
 
 
+class GroupProfileSerializer(serializers.Serializer):
+    """ Serializer for GroupProfile model interactions """
+    id = serializers.IntegerField(source='id')  # pylint: disable=C0103
```

Comment:
I think we don't need to explicitly declare fields here since `source` is same as name of field. 
```
class GroupProfileSerializer(serializers.ModelSerializer):
    class Meta:
        model = GroupProfile
        fields = ('id', 'name', )
```